test/test-functions.json

```
@@ -4,7 +4,7 @@
     {
       "src": "{horse :date}",
       "exp": "{|horse|}",
-      "errors": [{ "name": "RangeError" }]
+      "errors": [{ "type": "bad-input" }]
```

Our specification has a specific name for this. It is _Operand Mismatch Error_. We should use some formulation of that, perhaps:

```suggestion
      "errors": [{ "name": "OperandMismatchError" }]
```

---

That sounds like something to apply separately from this PR, for all the `type: 'bad-input'` errors.

---

Do you disagree with my observation?

Also, some "bad-inputs" might be syntax errors (I'd have to look to know).

---

I think the right context for that discussion in #706, because in the absence of a defined list of error codes, it's hard to say whether `bad-input`, `OperandMismatchError`, or maybe `operand-mismatch` is the right code to use.

If any of the current `bad-input` errors are indeed syntax errors, that should be fixed separately.

---

I think we should merge this small, limited fix-up PR in its current state, and work separately towards resolving #706 either iteratively or holistically.